__tests__/unit/handler-test.js

```diff
@@ -42,4 +42,8 @@ describe("Unit | handler", function () {
     expect(response.code).toBe(500);
     expect(response.data.errors[0].message).toBe("foo");
   });
+
+  it('supports creating handler with no options provided', function () {
+    expect(() => createGraphQLHandler(graphQLSchema, mirageSchema)).toThrow("Cannot destructure property `context` of 'undefined' or 'null'.")
+  })
```

Comment:
This should work:
```javascript
  it("supports creating handler with no options provided", function () {
    expect(() => createGraphQLHandler(graphQLSchema, mirageSchema)).not.toThrow(
      "Cannot destructure property `context` of 'undefined' or 'null'."
    );
  });
```

Prettier has some defaults and the above matches those conventions.

**Note**: I changed the assertion so it expects calling the function this way _doesn't_ throw.

Reply:
Whoops! I had `.not.` in there, but in my playing around to ensure the test was working accidentally took it out. Thanks for that catch!